src/main/scala/intellij/haskell/external/component/PathComponent.scala

```diff
@@ -0,0 +1,119 @@
+package intellij.haskell.external.component
+
+import java.io.File
+import java.net.URL
+
+import com.intellij.openapi.project.Project
+import com.intellij.util.io.URLUtil
+import intellij.haskell.HaskellNotificationGroup
+import intellij.haskell.external.commandLine.StackCommandLine
+import org.apache.commons.io.FileUtils
+
+import scala.io.Source
+
+object PathComponent {
+  private final val PackageNamePattern = """.* (.*) [==|installed].*""".r
+  private final val LocalPkgdbResolverPattern = """.*/(lts-\d+\.\d+)/.*\n""".r
+  private final val CabalConfigFileResolverPattern = """.*/(lts-\d+\.\d+)""".r
+
+  def getIdeaPath(project: Project): String = {
+    project.getBasePath + File.separator + ".idea"
+  }
+
+  def getConfigFilePath(project: Project): String = {
```

Comment:
It's better to put cabal.config file in project's folder. As also mentioned in cabal.config file.